.github/workflows/ci.yml

```
@@ -54,3 +54,14 @@ jobs:
           javac -J-Xmx32m -version
           mvn install -DskipTests=true -Dmaven.javadoc.skip=true -B -V
           mvn test -B
+  test-js:
+    name: Test JS
+    needs: build
+    if: |
+      !cancelled() && !failure()
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/setup-node@v4
+        with:
+          node-version: 20
+      - run: node src/test/javascript/salvation.test.js
```

```suggestion
      - run: node --test
```

It'll find files itself; that way we don't need to update this if we ever add another test file (which isn't that likely, but...)

---

Ah nice! #TIL